src/key.rs

```diff
@@ -502,7 +502,7 @@ pub fn verify_type(key_inner: *mut raw::RedisModuleKey, redis_type: &RedisType)
         let raw_type = unsafe { raw::RedisModule_ModuleTypeGetType.unwrap()(key_inner) };
 
         if raw_type != *redis_type.raw_type.borrow() {
-            return Err(RedisError::Str("Existing key has wrong Redis type"));
+            return Err(RedisError::Str("ERR Existing key has wrong Redis type"));
```

Comment:
1. I see several other similar errors without the `ERR` prefix
(searching for `Err(RedisError::Str(`)

2. Maybe we should have this `ERR` prefix added by `RedisError` (if not already present) and not hard-coded?
Similar to `impl<T: std::error::Error> From<T> for RedisError`